src/router/routes/admin/admin-checkout.ts

```diff
@@ -45,13 +45,25 @@ export class AdminCheckoutController extends AbstractScannerController implement
     // Get all checked out items
     app.get(
       '/',
-      this.authService.verifyAcl(this.scannerAcl, AclOperations.READ_ALL),
+      (req, res, next) => this.verifyScannerPermissionsMiddleware(
```

Comment:
Does this mean both scanners and team members can use this route, or that only scanners can?

Reply:
Both